schema.json

```diff
@@ -139,39 +145,44 @@
       },
       {
          "name": "mode_collecte",
-         "description": "Préciser les modalités de collecte des données : automatisé, comptages manuels, enquêtes, etc.",
-         "example": "Automatisé",
+         "description": "Préciser si la collecte des données est automatisée ou manuelle.",
+         "example": "automatisé",
          "type": "string",
          "constraints": {
             "required": true,
-            "enum": ["Manuel", "Automatisé", "Enquête", "Autre"]
+            "enum": ["manuel", "automatisé"]
          }
       },
       {
-         "name": "mode_calcul",
-         "description": "Façon dont est calculé ou agrégé l’indicateur à partir des données de base (ex : formule paramétrique ou pondération des données locales), en distinguant dans le cas d’un ratio, le numérateur et le dénominateur. Si nécessaire, fournir un exemple de calcul.",
-         "example": "Moyenne",
+         "name": "source_collecte",
+         "description": "Préciser comment la collecte est réalisée : script, enquête...",
+         "example": "Automatisé",
```

Comment:
Not sure that I understand this field, in regards to the previous one. The example should be updated maybe?

Reply:
The example is indeed wrong, but I'm not even sure about the field itself. There was the idea of changing the _mode_collecte_ field for only "manuel" or "automatisé" and adding a second field to specify how the data is collected, I don't really know how to formulate that.